addons/l10n_ro_edi_stock_batch/models/l10n_ro_edi_stock_etransport_interface.py

```diff
@@ -0,0 +1,59 @@
+from odoo import api, fields, models, _
+
+
+class L10nRoEdiStockETransportInterface(models.Model):
+    _inherit = 'l10n_ro.edi.stock.etransport.interface'
+
+    batch_id = fields.Many2one(comodel_name='stock.picking.batch', default=None)
+
+    # Compute Methods
+
+    @api.depends('picking_id.move_ids', 'batch_id.move_ids')
+    def _compute_l10n_ro_edi_stock_move_ids(self):
+        # Extends l10n_ro_edi_stock
```

Comment:
(applies for every "# Extends ..." comment you use in this file)
generally we use it like this: `# EXTENDS '<module_name>'`
But here, it should be `# OVERRIDES 'l10n_ro_edi_stock'`
because you are not calling  `super()._compute...` here